socorro/processor/processor_2015.py

```diff
@@ -207,27 +252,22 @@ def process_crash(self, raw_crash, raw_dumps, processed_crash):
         processed_crash.startedDateTime = processed_crash.started_datetime
         processed_crash.signature = 'EMPTY: crash failed to process'
 
-        crash_id = raw_crash.get('uuid', 'unknown')
+        crash_id = raw_crash['uuid']
         try:
             # quit_check calls ought to be scattered around the code to allow
             # the processor to be responsive to requests to shut down.
             self.quit_check()
 
-            processor_meta_data.started_timestamp = self._log_job_start(
-                crash_id
-            )
+            start_time = self.config.logger.info("starting job: %s", crash_id)
+            processor_meta_data.started_timestamp = start_time
 
-            # apply transformations
-            #    step through each of the rule sets to apply the rules.
-            for a_rule_set_name, a_rule_set in self.rule_system.iteritems():
-                # for each rule set, invoke the 'act' method - this method
-                # will be the method specified in fourth element of the
-                # rule set configuration list.
-                a_rule_set.act(
+            # apply_all_rules
+            for rule in self.rules:
+                predicate_result, action_result = rule.act(
```

Comment:
This line captures results in variables, but I don't see it doing anything with them. Did you intend to do something with them? If not, I think it's better not to capture them. Plus I think it'll make the linter sad.

Reply:
good catch. these were used in a previous context. not sure why the linter didn't catch it? fixed in the next commit.